Jenkinsfile

```diff
@@ -9,12 +9,22 @@ elifePipeline {
             }
 
             stage 'Build image', {
-                dockerBuild 'api-dummy', commit
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
             }
 
             stage 'Project tests', {
-                dockerBuildCi 'api-dummy', commit
-                dockerProjectTests 'api-dummy', commit
+                try {
```

Comment:
tris `try`+`finally` should be exactly what https://github.com/elifesciences/elife-jenkins-workflow-libs/blob/master/vars/dockerProjectTests.groovy does, why doesn't it work here?